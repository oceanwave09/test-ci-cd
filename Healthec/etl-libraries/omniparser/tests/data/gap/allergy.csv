~Row~,~Domain~,~Patient First Name~,~Patient Last Name~,~Patient Date of Birth~,~External Patient ID~,~External Practice ID~,~Code~,~Name~,~Dictionary Name~,~Allergy Start Date~,~Allergy End Date~,~Allergy Reaction Code~,~Allergy Reaction Name~,~Allergy Reaction Dictionary~,~History Flag~,~Vendor~,~Source Type~,~PATIENT_ID~
~1~,~Allergies~,~First7660601~,~Last7660601~,~19600101~,~7660601~,~Epic~,~7129~,~LISINOPRIL~,~~,~20190829~,~~,~1046~,~Other~,~Beaumont_Epic_Allergy_Reaction~,~~,~~,~~,~RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2~
~2~,~Allergies~,~First7660602~,~Last7660602~,~19580101~,~7660602~,~Epic~,~1000003~,~SEASONAL ALLERGIES~,~~,~20220907~,~~,~6~,~Runny Nose~,~Beaumont_Epic_Allergy_Reaction~,~~,~~,~~,~IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R~
~3~,~Allergies~,~First7660603~,~Last7660603~,~19920101~,~7660603~,~Epic~,~33~,~SULFA ANTIBIOTICS~,~~,~~,~~,~1022~,~Hives~,~Beaumont_Epic_Allergy_Reaction~,~~,~~,~~,~DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF~
~4~,~Allergies~,~First7660604~,~Last7660604~,~19890101~,~7660604~,~Epic~,~33~,~SULFA ANTIBIOTICS~,~~,~20141209~,~~,~~,~~,~Beaumont_Epic_Allergy_Reaction~,~~,~~,~~,~S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV~
~5~,~Allergies~,~First7660605~,~Last7660605~,~19690101~,~7660605~,~Epic~,~14~,~MORPHINE AND RELATED~,~~,~20141022~,~~,~2~,~Hives~,~Beaumont_Epic_Allergy_Reaction~,~~,~~,~~,~NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU~
