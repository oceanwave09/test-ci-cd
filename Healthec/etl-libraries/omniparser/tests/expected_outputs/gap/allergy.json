[
	{
		"allergy_reaction_code": "1046",
		"allergy_reaction_dictionary": "Beaumont_Epic_Allergy_Reaction",
		"allergy_reaction_name": "Other",
		"allergy_start_date": "20190829",
		"code": "7129",
		"domain": "Allergies",
		"external_patient_id": "7660601",
		"external_practice_id": "Epic",
		"name": "LISINOPRIL",
		"patient_dob": "b699c2b9c5b08522dc986a2def4aca6e",
		"patient_first_name": "920b1bfa8aeb6237faf53f11643c8ecf",
		"patient_id": "RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2",
		"patient_last_name": "41c337237eef08cdc7d9f625231d112a",
		"row": "1",
		"row_id": "59a8b1f9-1ab4-4904-ad92-ff93d3702546"
	},
	{
		"allergy_reaction_code": "6",
		"allergy_reaction_dictionary": "Beaumont_Epic_Allergy_Reaction",
		"allergy_reaction_name": "Runny Nose",
		"allergy_start_date": "20220907",
		"code": "1000003",
		"domain": "Allergies",
		"external_patient_id": "7660602",
		"external_practice_id": "Epic",
		"name": "SEASONAL ALLERGIES",
		"patient_dob": "56a973e9f56eb206d960f2974ef9804b",
		"patient_first_name": "4369a95df3b5152bdd7c64a72efd1aa6",
		"patient_id": "IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R",
		"patient_last_name": "fc65ea507d571ca1dde7d800f2f95573",
		"row": "2",
		"row_id": "3c02a3b4-b43a-43f4-85de-f5d8d57fb7bb"
	},
	{
		"allergy_reaction_code": "1022",
		"allergy_reaction_dictionary": "Beaumont_Epic_Allergy_Reaction",
		"allergy_reaction_name": "Hives",
		"code": "33",
		"domain": "Allergies",
		"external_patient_id": "7660603",
		"external_practice_id": "Epic",
		"name": "SULFA ANTIBIOTICS",
		"patient_dob": "65d8491e60430a0736b2abfd79856d4d",
		"patient_first_name": "0430717827ce4c57e07c1e445bc70477",
		"patient_id": "DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF",
		"patient_last_name": "bd39484c0baafaf573a91d7eeb470750",
		"row": "3",
		"row_id": "fcee3f23-811e-411a-bab2-e9ee2761db59"
	},
	{
		"allergy_reaction_dictionary": "Beaumont_Epic_Allergy_Reaction",
		"allergy_start_date": "20141209",
		"code": "33",
		"domain": "Allergies",
		"external_patient_id": "7660604",
		"external_practice_id": "Epic",
		"name": "SULFA ANTIBIOTICS",
		"patient_dob": "9fde4afa3f81b3e57465ff6668b68b41",
		"patient_first_name": "4c2170080510d285b48e195f9c690c5a",
		"patient_id": "S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV",
		"patient_last_name": "29244f7cbc96db65581dba3839e472fe",
		"row": "4",
		"row_id": "a7e0a255-6081-40fe-a26b-22809bf86021"
	},
	{
		"allergy_reaction_code": "2",
		"allergy_reaction_dictionary": "Beaumont_Epic_Allergy_Reaction",
		"allergy_reaction_name": "Hives",
		"allergy_start_date": "20141022",
		"code": "14",
		"domain": "Allergies",
		"external_patient_id": "7660605",
		"external_practice_id": "Epic",
		"name": "MORPHINE AND RELATED",
		"patient_dob": "6d65a93dd15e1ce35deca05d2f90426c",
		"patient_first_name": "fe7030703fd7abee786ea532bf04cd7a",
		"patient_id": "NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU",
		"patient_last_name": "4b4e3c02a281e755267bd96cc57327b0",
		"row": "5",
		"row_id": "d9c48662-caa0-49d5-9d98-3be4af6457af"
	}
]
